extractor/text.go

```diff
@@ -240,7 +245,8 @@ func (e *Extractor) extractPageText(contents string, resources *model.PdfPageRes
 					return err
 				}
 				err = to.setFont(name, size)
-				if err != nil {
+				to.invalidFont = errors.Is(err, core.ErrNotSupported)
```

Comment:
go 1.12 compatibility.. need to stick with it
Could use 
https://godoc.org/golang.org/x/xerrors
in the meantime which provides some of this functionality?